.pre-commit-config.yaml

```diff
@@ -11,3 +11,10 @@ repos:
         entry: python conda/requirements.py
         language: python
         files: ^(requirements\.txt|pyproject\.toml)$
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.13.0
+    hooks:
+    -   id: mypy
+        args: []
```

Comment:
I think this may be removed
```suggestion
```

Reply:
It's needed to override the one [set in the recipe repo](https://github.com/pre-commit/mirrors-mypy/blob/main/.pre-commit-hooks.yaml#L7C5-L7C64)

I prefer to configure mypy using pyproject.toml so we can easily run it manually